docs/team/chiageng.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: Chia Geng's Project Portfolio Page
+---
+
+### Project: PoochPlanner
+
+PoochPlanner is a desktop application to track details of various groups (vendors, staff) that dog cafe owners have to regularly interact with. The app is optimised for use using Command Line Interface (CLI) while still encompassing a user-friend Graphical User Interface (GUI).
+
+Given below are my contributions to the project.
+
+* **New Feature**: Added the ability to add contact.
+    * What it does: allows the user to add a new contact.
+    * Justification: This feature improves the product significantly because a user can store all contacts in this address book.
+    * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
+    * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+
+* **Code contributed**: [RepoSense link]()
+
+* **Documentation**:
+    * User Guide:
+        * Added documentation for the features `add` [\#72]()
+    * Developer Guide:
+        * Added non functional requirements.
+        * Updated use cases for PoochPlanner
```

Comment:
standardise either ending with full stop or without full stop

Reply:
Thank you for noticing this